modules/parquet/test/stubs.js

```diff
@@ -0,0 +1,568 @@
+/* eslint-disable camelcase */
+
+export const ALL_TYPES_DICTIONARY_STUB = [
```

Comment:
Not sure if calling these `_EXPECTED` instead of `_STUB` would be more clear?

Reply:
Renamed.